tools/version.props.template

```diff
@@ -0,0 +1,14 @@
+<Project>
+  <PropertyGroup>
+    <Version>@Version</Version>
+    <Authors>Henrik Feldt, Logibit AB, formerly @mausch</Authors>
+    <Company>Steve Hatchett (Ported to .NET Standard by Theodore Tsirpanis)</Company>
+    <Copyright>Copyright © 2013-2017 SoftWx, Inc.</Copyright>
+    <PackageLicenseUrl>http://opensource.org/licenses/MIT</PackageLicenseUrl>
+    <PackageTags>testing fsharp assert expect</PackageTags>
+    <PackageIconUrl>https://raw.githubusercontent.com/haf/expecto/master/docs/expecto-logo-small.png</PackageIconUrl>
+    <RepositoryUrl>https://github.com/haf/expecto</RepositoryUrl>
+    <PackageProjectUrl>https://github.com/haf/expecto</PackageProjectUrl>
+    <Description>Expecto is a smooth test framework for F#, cloned from Fuchu with added functionality for making it easier to use.</Description>
```

Comment:
This is the common description shared by all three packages. A specific description for each package will be written.